src/pages/Accounts/Current/Current.tsx

```diff
@@ -15,21 +16,36 @@ import {
   CheckboxComponent,
   ButtonConmponent,
 } from '../../../components';
-
+import { requestForCurrentAccount } from '../../../redux/actions/Current';
```

Comment:
Add this export requestForCurrentAccount in the index file inside actions and then import from the index.

Reply:
changes updated